common/web/recorder/src/index.ts

```diff
@@ -618,7 +620,7 @@ export class RecordedSequenceTestSet implements TestSet<RecordedKeystrokeSequenc
     let testSet = this.testSet;
 
     for(var i=0; i < testSet.length; i++) {
-      var testSeq = this[i];
+      var testSeq = this.testSet[i];
```

Comment:
Is this a bug fix?

Reply:
Yes.  Once I saw it, I started wondering how (or even, if) it worked before.  (Same as the prior comment)